README.md

```diff
@@ -1603,7 +1603,7 @@ Update Time, five active automations, webhooks.
 ## Design Inspiration
 
   * [awwwards.](https://www.awwwards.com/) - [Top websites] A showcase of all the best-designed websites (voted on by designers).
-  * [Websitevise](https://websitevice.com/) - [Design showcase] A curated gallery of professional website designs to inspire developers and designers seeking creative ideas and best practices.
+  * [Websitevice](https://websitevice.com/) - [Design showcase] A curated gallery of professional website designs to inspire developers and designers seeking creative ideas and best practices.
```

Comment:
This is just links to other sites, think I'll pass.

The other design showcases have screenshots etc which at least takes some effort to host etc, this is just a link farm.

Reply:
Okay thank you for the review.